generators/client-2/templates/src/main/webapp/app/account/_account.ng2module.ts

```diff
@@ -80,6 +81,7 @@ let ACCOUNT_STATES = [
         SettingsComponent
     ],
     providers: [
+        CookieService,
```

Comment:
Since it's declared in shared module already it's not required here.  Can you remove it with another pr